src/components/calculator.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+
+export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <div className="calculator__wrap">
+          <div className="calculator__display">
+            0
+          </div>
+          <div className="calculatorUI">
+            <button type="button" value="clear">AC</button>
+            <button type="button" value="pos-neg">+/-</button>
+            <button type="button" value="%">%</button>
+            <button type="button" value="/" className="btn--opaque">/</button>
```

Comment:
- [ ] Let us stick to the design. Kindly use the &div; symbol, instead of this one as it would come in handy in subsequent projects.

Reply:
[Issue solved](https://github.com/Wahaj-Ali/maths-magicians/commit/07785313acf98660d8091ece3f68fe8740828ae7)